File: projektmanagement/src/main/java/com/szoftverhazi/projektmanagement/dto/TaskDto.java
package com.szoftverhazi.projektmanagement.dto;

import com.szoftverhazi.projektmanagement.enums.TaskStatus;
import com.szoftverhazi.projektmanagement.enums.TaskType;
import lombok.Data;

import java.util.Date;
@Data
public class TaskDto {
    private Long id;
    private String title;
    private TaskType type;
    private TaskStatus status;
    private Date dueDate;
    private Long employeeId;
    private String employeeName;

}

File: frontend/src/app/models/task.dto.ts
export class TaskDto {
    id!: number;
    title!: string;
    type!: string; // E.g. 'FELADAT', 'MEETING' - lehet enum
    status!: string; // E.g. 'FOLYAMATBAN', 'TELJESÍTVE', 'ELHALASZTVA' - lehet enum
    dueDate!: Date;
    employeeId!: number; // A felelős felhasználó ID-je
    employeeName!: string; // A felelős felhasználó neve
    projectId!: number; // A projekt, amelyhez tartozik a feladat
  }
  